src/components/clusters/ClusterDetailsMultiRegion/components/MachinePools/components/EditMachinePoolModal/fields/WindowsLicenseIncludedField.tsx

```diff
@@ -10,12 +10,11 @@ import ExternalLink from '~/components/common/ExternalLink';
 import PopoverHint from '~/components/common/PopoverHint';
 import { MachineType, NodePool } from '~/types/clusters_mgmt.v1';
 
-const fieldId = 'windowsLicenseIncluded';
+const fieldId = 'isWindowsLicenseIncluded';
 
 type WindowsLicenseIncludedFieldProps = {
   isEdit?: boolean;
-  // Manually adding this field until backend api adds support to it -> https://issues.redhat.com/browse/OCMUI-2905
-  currentMP?: NodePool & { imageType?: string };
+  currentMP?: NodePool;
```

Comment:
Nice that we implemented everything before even types where published and now it's only a matter of removing a few things and everything works :sparkles: